### PR TITLE
Feature/kitosudv 1933 kle

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.ApplicationServices/KLE/IKLEApplicationService.cs [hunk 1]
```diff
@@ -10,5 +15,13 @@ public interface IKLEApplicationService
         Result<KLEStatus, OperationFailure> GetKLEStatus();
         Result<IEnumerable<KLEChange>, OperationFailure> GetKLEChangeSummary();
         Result<KLEUpdateStatus, OperationFailure> UpdateKLE(int organizationId);
+        /// <summary>
+        /// Performs KLE search and responds with a versioned result set
+        /// </summary>
+        /// <param name="criteria"></param>
+        /// <returns></returns>
+        Result<(Maybe<DateTime> updateReference, IQueryable<TaskRef> contents), OperationError> SearchKle(params IDomainQuery<TaskRef>[] criteria);
```
Comment (JacobMalling) [inline Core.ApplicationServices/KLE/IKLEApplicationService.cs:23]: Nogen grund til at det er "criteria" og ikke "conditions" som de andre lignende services?

File: Core.DomainServices/Repositories/TaskRefs/ITaskRefRepository.cs [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+using System.Linq;
 using Core.DomainModel.Organization;
+using Core.DomainServices.Queries;
 using Infrastructure.Services.Types;
 
 namespace Core.DomainServices.Repositories.TaskRefs
 {
     public interface ITaskRefRepository
     {
+        IQueryable<TaskRef> Query(params IDomainQuery<TaskRef>[] criteria);
```
Comment (JacobMalling) [inline Core.DomainServices/Repositories/TaskRefs/ITaskRefRepository.cs:11]: Nogen grund til at det er "criteria" og ikke "conditions" som de andre lignende services?

File: Tests.Unit.Presentation.Web/Tests.Unit.Presentation.Web.csproj [hunk 3]
```diff
@@ -287,6 +291,7 @@
       <Name>Tests.Toolkit</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup />
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Tests.Unit.Presentation.Web.csproj:294]: Denne ser ud til at være en fejl

**Review by JacobMalling — APPROVED** [review]
Det ser jo meget godt ud

File: Tests.Unit.Presentation.Web/Tests.Unit.Presentation.Web.csproj (hunk 3 above)
Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Tests.Unit.Presentation.Web.csproj:294]: ja sådan er det nogengange med de merges der. De efterlader noget rod, men det fixer jeg

File: Core.DomainServices/Repositories/TaskRefs/ITaskRefRepository.cs (hunk 2 above)
Comment (mrjsawdk) [reply Core.DomainServices/Repositories/TaskRefs/ITaskRefRepository.cs:11]: næh. jeg renamer

**Pull request merged**